### PR TITLE
Completes #52 - Ability to Sort List of Value Sets by attributes

### PR DESCRIPTION
**Pull request by rustyswayne**

### CODE REVIEW

File: Fabric.Terminology.IntegrationTests/Services/ClientTermValueSetServiceTests.cs [hunk 1]
```diff
@@ -95,31 +95,30 @@ public void CanRemoveCodes(string name, int initialCodeCount, int removeCodeCoun
             this.clientTermValueSetService.Delete(result);
         }
 
+        // TODO fix
```
Comment (corytak) [inline Fabric.Terminology.IntegrationTests/Services/ClientTermValueSetServiceTests.cs]: Just making sure that this TODO is something for future edits and not something that you wanted to have fixed for this pull request.

File: Fabric.Terminology.SqlServer/Persistence/SqlValueSetCodeRepository.cs [hunk 2]
```diff
@@ -25,17 +26,21 @@ internal class SqlValueSetCodeRepository : IValueSetCodeRepository
 
         private readonly IPagingStrategyFactory pagingStrategyFactory;
 
+        private readonly IOrderingStrategyFactory orderingStragetyFactory;
```
Comment (corytak) [inline Fabric.Terminology.SqlServer/Persistence/SqlValueSetCodeRepository.cs]: There's a typo here.

File: Fabric.Terminology.SqlServer/Persistence/SqlValueSetCodeRepository.cs [hunk 3]
```diff
@@ -183,18 +188,21 @@ private IReadOnlyCollection<IValueSetCode> QueryValueSetCodes(IEnumerable<Guid>
...
+            source = orderingStrategy.SetOrdering(source, pagerSettings.AsOrderingParameters());
+
+            var items = await source.Skip((pagerSettings.CurrentPage - 1) * pagerSettings.ItemsPerPage)
                             .Take(pagerSettings.ItemsPerPage)
                             .ToListAsync();
 
             var factory = new ValueSetCodeFactory();
 
-            // TODO can't cache at this point since codeGuid is null in db.  Fixme
+            // TODO can't cache at this point since codeGuid is null in db.  Fix me
```
Comment (corytak) [inline Fabric.Terminology.SqlServer/Persistence/SqlValueSetCodeRepository.cs]: Ensuring this TODO is referring to a future todo and not something you wanted to have completed in this PR

File: Fabric.Terminology.IntegrationTests/Services/ClientTermValueSetServiceTests.cs (hunk 1 above)
Comment (rustyswayne) [reply Fabric.Terminology.IntegrationTests/Services/ClientTermValueSetServiceTests.cs]: This class is actually excluded from the solution at the moment - and is something that needs to be refactored in it's entirety in the future.  The tests fell apart with the changes we made for with the model used for Adding and Updating value sets as now the Codes cannot be simply made up - they have to have a valid CodeGuid ...

**Pull request merged**